### PR TITLE
VTC_001 innit

### PR DESCRIPTION
**Pull request by Ghostyshka**
VTC_001: make a solution, create a separate project with an approach to the database, add a framework to the entity

### CODE REVIEW

File: VetClinicManagament/VetClinicManagament/Program.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+//builder.Services.AddDbContext<DataContext>(options =>
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicManagament/Program.cs:12]: please remove commented code

File: VetClinicManagament/VetClinicManagament.sln [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.10.34928.147
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VetClinicManagamentApi", "VetClinicManagament\VetClinicManagamentApi.csproj", "{14BF8F90-CF4E-4403-AD04-AF008CF05AEF}"
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicManagament.sln:6]: please use naming convention e.g.
VetClinicManagament.Api
VetClinicManagament.Core
VetClinicManagament.Domain

File: VetClinicManagament/VetClinicManagament/VetClinicManagament.http [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@VetClinicManagament_HostAddress = http://localhost:5228
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicManagament/VetClinicManagament.http:1]: Why do you need it?

File: VetClinicManagament/VetClinicManagament/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "ConnectionStrings": {
+        "DefaultConnection": "Host=localhost;Database=VetClinics;Username=postgres;Password=admin"  },
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicManagament/appsettings.json:3]: remove it from here and add this value to the appsettings.Development.json

File: VetClinicManagament/VetClinicsDomain/Data/DataContext.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace VetClinicsDomain.Data;
+
+public class DataContext : DbContext
+{
+    public DataContext(DbContextOptions<DataContext> options) : base(options)
+    {
+        
+    }
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicsDomain/Data/DataContext.cs:12]: no need for it anymore, please remove

File: VetClinicManagament/VetClinicsDomain/Entity/Animal.cs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+﻿namespace VetClinicsDomain.Entity;
+
+public class Animal
+{
+    public int AnimalId { get; set; }
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicsDomain/Entity/Animal.cs:5]: WDYT about animal types like dog, cat, parrot, etc.?

File: VetClinicManagament/VetClinicsDomain/Entity/User.cs [hunk 7]
```diff
@@ -0,0 +1,16 @@
+﻿namespace VetClinicsDomain.Entity;
+
+public class User
+{
+    public int UserId { get; set; }
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicsDomain/Entity/User.cs:5]: should be some relationship between user and animals (many to many)

File: VetClinicManagament/VetClinicsDomain/Entity/User.cs [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    public string FirstName { get; set; } = string.Empty;
+    public string LastName { get; set; } = string.Empty;
+    public int DateOfBirth { get; set; }                // TO DO: implement happy birthday notifications
```
Comment (denyshfive) [inline VetClinicManagament/VetClinicsDomain/Entity/User.cs:8]: int?

File: VetClinicManagament/VetClinicsDomain/Entity/Animal.cs (hunk 6 above)
Comment (denyshfive) [reply VetClinicManagament/VetClinicsDomain/Entity/Animal.cs:5]: Also others characteristics

**Review by denyshfive — CHANGES_REQUESTED** [review]

File: VetClinicManagament/VetClinicManagament/VetClinicManagament.http (hunk 3 above)
Comment (Ghostyshka) [reply VetClinicManagament/VetClinicManagament/VetClinicManagament.http:1]: For local testing of requests, here is an example of how it works. I don't have an actual request to demonstrate, but this serves as a working example:
image

File: VetClinicManagament/VetClinicsDomain/Entity/User.cs (hunk 8 above)
Comment (Ghostyshka) [reply VetClinicManagament/VetClinicsDomain/Entity/User.cs:8]: replaced as DateTime, thx

File: VetClinicManagament/VetClinicsDomain/Entity/Animal.cs (hunk 6 above)
Comment (Ghostyshka) [reply VetClinicManagament/VetClinicsDomain/Entity/Animal.cs:5]: updated Animal entity

File: VetClinicManagament/VetClinicsDomain/Entity/User.cs (hunk 7 above)
Comment (Ghostyshka) [reply VetClinicManagament/VetClinicsDomain/Entity/User.cs:5]: refactored

**Pull request merged**